tests/test_project_management.py

```diff
@@ -0,0 +1,222 @@
+OPERATION_IN_PROGRESS_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': False
+})
+OPERATION_FAILED_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'error': 'some error',
+})
+ANDROID_APP_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_ANDROID_APP,
+})
+ANDROID_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_ANDROID_APP_NO_DISPLAY_NAME,
+})
+IOS_APP_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_IOS_APP,
+})
+IOS_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE = json.dumps({
+    'name': OPERATION_NAME,
+    'done': True,
+    'response': TEST_IOS_APP_NO_DISPLAY_NAME,
+})
+
+ERROR_RESPONSE = 'some error'
+
+class TestCreateAndroidApp(object):
+    _CREATION_URL = '{0}/v1beta1/projects/{1}/{2}'.format(BASE_URL, TEST_PROJECT_ID, "androidApps")
+    _POLLING_URL = '{0}/v1/{1}'.format(BASE_URL, OPERATION_NAME)
+
+    @classmethod
+    def setup_class(cls):
+        firebase_admin.initialize_app(
+            testutils.MockCredential(), {'projectId': TEST_PROJECT_ID})
+
+    @classmethod
+    def teardown_class(cls):
+        testutils.cleanup_apps()
+
+    def _set_up_mock_responses_and_request_captor_for_project_management_service(
+            self, statuses, responses, app=None):
+        if not app:
+            app = firebase_admin.get_app()
+        project_management_service = project_management._get_project_management_service(app)
+        captor = []
+        project_management_service._client.session.mount(
+            'https://firebase.googleapis.com',
+            testutils.MockMultiRequestAdapter(responses, statuses, captor))
+        return captor
+
+    def test_create_android_app_without_display_name(self):
+        captor = self._set_up_mock_responses_and_request_captor_for_project_management_service(
+            statuses=[200, 200, 200],
+            responses=[
+                OPERATION_IN_PROGRESS_RESPONSE,  # Request to create Android app asynchronously.
+                OPERATION_IN_PROGRESS_RESPONSE,  # Creation Operation is still not done.
+                ANDROID_APP_NO_DISPLAY_NAME_OPERATION_SUCCESSFUL_RESPONSE,  # Operation completed.
+            ])
+
+        android_app = project_management.create_android_app(
+            package_name=TEST_ANDROID_APP_PACKAGE_NAME)
+
+        assert android_app.app_id == TEST_ANDROID_APP_ID
+        assert len(captor) == 3
+        assert captor[0].method == 'POST'
+        assert captor[0].url == TestCreateAndroidApp._CREATION_URL
+        body = {'packageName': TEST_ANDROID_APP_PACKAGE_NAME}
+        assert json.loads(captor[0].body.decode()) == body
+        assert captor[1].method == 'GET'
+        assert captor[1].url == TestCreateAndroidApp._POLLING_URL
+        assert not captor[1].body
```

Comment:
Feels kind of unnecessary for a GET request.

Reply:
While it's true that the integration test will result in a HTTP 400 failure if we ever passed a body to this `GET` request, I think it's good for the unit test to catch this error as well. For example, the `json=request_body` parameter could result from a copy-paste error.